app/services/precheck_eligibility_service.rb

```diff
@@ -37,6 +37,10 @@ def actionable_errors
     end
   end
 
+  def too_late_or_checked_in?
```

Comment:
As a personal practice I like to add at least one test for any public method I add, but I can also see why it might feel unnecessary in this case!